applications/nrf_desktop/doc/smp.rst

```diff
@@ -28,27 +28,28 @@ Configuration
 
 Make sure that the following Kconfig options are enabled:
 
-* :option:`CONFIG_MCUMGR_CMD_IMG_MGMT`
+* ``CONFIG_MCUMGR_CMD_IMG_MGMT``
   The module uses MCUmgr image management handlers.
   See the :ref:`zephyr:device_mgmt` for details about Zephyr's device management subsystem.
-* :option:`CONFIG_MCUMGR_SMP_BT`
-  Device receives SMP commands over Bluetooth.
+* ``CONFIG_MCUMGR_SMP_BT``
+  The device receives SMP commands over Bluetooth.
 * ``CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE``
-  The :ref:`nrf_desktop_dfu` performs secondary image FLASH area erase in the background and confirms the currently running MCUboot image.
-* :option:`CONFIG_BOOTLOADER_MCUBOOT`
-  The DFU over Simple Management Protocol in Zephyr is supported only with MCUboot bootloader.
+  The :ref:`nrf_desktop_dfu` performs a secondary image FLASH area erase in the background and confirms the currently running MCUboot image.
+* ``CONFIG_BOOTLOADER_MCUBOOT``
+  The DFU over Simple Management Protocol in Zephyr is supported only with the MCUboot bootloader.
 
-Enable the module with ``CONFIG_DESKTOP_SMP_ENABLE`` Kconfig option.
+Enable the module using the ``CONFIG_DESKTOP_SMP_ENABLE`` Kconfig option.
 
 Implementation details
 **********************
 
-During initialization, the module registers the SMP Bluetooth service.
+During the initialization, the module registers the SMP Bluetooth service.
 
 The module registers the :cpp:func:`upload_confirm` callback that is used to confirm or reject the image upload.
 The image upload is rejected only in case the :ref:`nrf_desktop_dfu` has acquired the ``dfu_lock`` (:file:`src/util/dfu_lock.h`).
-If image is being received through the SMP, the Simple Management Protocol application module acquires the lock and firmware update through the :ref:`nrf_desktop_dfu` is disabled until nRF Desktop device is rebooted.
-Modifying the data in the secondary image slot by both modules would result in broken firmware image that would be rejected by the bootloader.
+If the image is being received through the SMP, the Simple Management Protocol application module acquires the lock.
+Also, firmware update through the :ref:`nrf_desktop_dfu` is disabled until the nRF Desktop device is rebooted.
+Any modification of the data in the secondary image slot by both modules would result in a broken firmware image that would be rejected by the bootloader.
 
 The module periodically submits the ``ble_smp_transfer_event`` while the image is being uploaded.
-This event is used by :ref:`nrf_desktop_ble_latency` that decreases the connection slave latency to speed up the transfer.
+This event is used by :ref:`nrf_desktop_ble_latency` to start decreasing the connection slave latency to speed up the transfer.
```

Comment:
It's more like: `ble_latency` module is informed that the slave latency needs to be decreased when it receives this event. I rephrased this.